experiments/language_model/mlm.sh

```diff
@@ -82,11 +85,14 @@ esac
 python -m DeBERTa.apps.run --model_config config.json  \
 	--tag $tag \
 	--do_train \
-	--num_training_steps 1000000 \
+	--use_mpi 1 \
+	--num_training_steps 1000 --warmup_proportion 0.1 \
```

Comment:
Warmup proportion only adjusted here to be able to use lower number of training steps as an example. Else warmup steps are more than 1000 so it crashes.